src/components/MultiStepIndicator/components/StepsDivider/StepsDivider.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import "./StepsDivider.scss";
+
+const StepsDivider = ({ dividerComponentClassName }) => {
+  return <div className={cx("monday-style-steps-divider-component", dividerComponentClassName)} />;
```

Comment:
We have a divider why create another?

Reply:
Because for some reason, creating a separate component seemed easier at the time than simply passing a classname prop 🤦